### DISCUSSION TITLE
multiselect dropdown in custom fields

### DISCUSSION THREAD

**Discussion by Husnain-QH** [kimai/kimai#3471] [General]
How can I add multiselect dropdown using the custom fields plugin or any other way to Activities form?

**Comment by kevinpapst** [comment]
If you write your own plugin, it is easily possible
If you use Custom-Fields plugin, you have to wait for the next major release #2902

**Reply by Husnain-QH** [reply]
We are using Custom Fields plugin, would like to learn more about defining custom fields through code, unfortunately even after reading the documentation many times, I was unable to understand how to add a custom field without the custom fields plugin, would really appreciate help on adding a multiselect field on the Activities Definition page.

**Reply by kevinpapst** [reply]
I cannot help with the part of "understanding an event listener" in the context of a Symfony bundle.
But the DemoBundle has a working example: https://github.com/Keleo/DemoBundle/blob/master/EventSubscriber/MetaFieldSubscriber.php

If you have code questions, feel free to share them.

**Reply by Husnain-QH** [reply]
By when will this release "https://github.com/kevinpapst/kimai2/pull/2902" be available?
